db/generateSeedData.js

```diff
@@ -192,6 +191,9 @@ let generateAnswers = (num, numQuestions) => {
     return answersArr;
 }
 
+
+
+
 module.exports.generateItems = generateItems;
```

Comment:
Can change `module.exports.some_exported_object_or_function` to: 
`module.exports = {
  generateItems,
  generateReviews,
  generateFrequentlyTogether,
}`